src/how-it-works.html

```diff
@@ -2,76 +2,114 @@
     </div>
+  </div>
+
+  <div class="layout-grid margin-bottom main-container main-container--narrow">
     <div class="flex-container">
+      <h3><span class="colored-text">Skip</span> that song</h3>
+      <p>
+        Entertaining your parrots is now a piece of cake. BeakBox allows your birds to choose what they want to listen to. If a tune doesn't quite hit the mark, they can skip to the next one with just
+        one push.
+      </p>
+    </div>
+    <img src="/imgs/blurb-bird.svg" alt="" />
+  </div>
+
+  <div class="layout-grid margin-bottom main-container main-container--narrow">
+    <img src="/imgs/blurb-beakbox-side.svg" alt="" />
+
+    <div class="flex-container">
+      <h3>DJ your <span class="colored-text">parrot's day</span></h3>
+      <p>Easily upload your bird's favorite tunes and create playlists for different moods. Watch as your parrots groove to the beats.</p>
+    </div>
+  </div>
+
+  <div class="layout-grid padding-bottom main-container main-container--narrow">
+    <div class="flex-container">
+      <h3>Training, bonding, and <span class="colored-text">beyond</span></h3>
+      <p>Train your parrots to learn new tricks, unlearn destructive behaviors, and strengthen your bond in the process.</p>
+    </div>
+
+    <img src="/imgs/blurb-scholar-bird.svg" alt="" />
+  </div>
+</section>
+
+<div class="divider-container-bottom">
+  <svg id="wave" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2678.5 180">
+    <path
+      d="M0,0H2678.5V180l-18.64-6c-18.14-6-55.82-18-92.69-42-37.28-24-74.76-60-111.84-72-37.28-12-74.46,0-111.84,0s-74.25-12-111.84-21c-37.28-9-74.05-15-110.83-15s-74.76,6-111.84,12c-37.28,6-74.56,12-111.84,15s-74.36,3-111.84,21c-37.28,18-74.05,54-110.83,72-37.28,18-74.86,18-111.84,9-37.28-9-74.66-27-111.84-45-37.28-18-74.36-36-111.84-36s-74.15,18-111.84,39c-36.27,21-73.95,45-110.83,48-37.28,3-74.76-15-111.84-15s-74.46,18-111.84,3c-37.28-15-74.25-63-111.84-63s-74.05,48-110.83,48-74.76-48-111.84-75c-37.28-27-74.56-33-111.84-9-37.28,24-74.36,78-111.84,99-37.28,21-74.05,9-110.83,0-37.28-9-74.86-15-111.84-24-37.28-9-74.66-21-93.2-27L0,90V0Z"
+      fill="#feeeea"
+    />
+  </svg>
+</div>
+
+<section>
+  <div class="main-container main-container--align-center flex-container">
+    <h2>See BeakBox <span class="colored-text">in action</span></h2>
+    <div class="flex-container-collapse">
       <div>
         <iframe
           width="560"
           height="315"
-          loading="lazy"
-          src="https://www.youtube.com/embed/1y3ofZExm60"
-          title="YouTube video showing Kea using the BeakBox"
+          src="https://www.youtube.com/embed/BOR2ctjOflc?si=6jXtRPEh9mIMPbR3"
+          title="YouTube video player"
           frameborder="0"
-          allow="autoplay; clipboard-write; encrypted-media;picture-in-picture; web-share"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
```

Comment:
I would look in to all the values in this list. It looks like google is trying to fingerprint and track visitors that view this page and iframe.